examples/SharedMemory/SharedMemoryCommands.h

```diff
@@ -523,18 +523,19 @@ struct LoadSoftBodyArgs
 	double m_mass;
 	double m_collisionMargin;
 	double m_initialPosition[3];
-        double m_initialOrientation[4];
-        double m_springElasticStiffness;
-        double m_springDampingStiffness;
-        double m_corotatedMu;
-        double m_corotatedLambda;
-        int m_useBendingSprings;
-        double m_collisionHardness;
-        double m_useSelfCollision;
-        double m_frictionCoeff;
-        double m_NeoHookeanMu;
-        double m_NeoHookeanLambda;
-        double m_NeoHookeanDamping;
+    double m_initialOrientation[4];
+    double m_springElasticStiffness;
+    double m_springDampingStiffness;
+    double m_springBendingStiffness;
```

Comment:
If you insert a variable in the middle, you need to update the version at the top:
https://github.com/bulletphysics/bullet3/blob/7bce5d61f485d8f17c3c7d0400b550ff4e0e4822/examples/SharedMemory/SharedMemoryPublic.h#L11

Reply:
Also, please fix your editor for next CL to use tabs instead of spaces. I have to run the clang-format soon, because of all the spaces of your commits. (I'll do this cleanup in a separate PR that doesn't contain any code changes)

Reply:
> If you insert a variable in the middle, you need to update the version at the top:
> https://github.com/bulletphysics/bullet3/blob/7bce5d61f485d8f17c3c7d0400b550ff4e0e4822/examples/SharedMemory/SharedMemoryPublic.h#L11

Done.

Reply:
> Also, please fix your editor for next CL to use tabs instead of spaces. I have to run the clang-format soon, because of all the spaces of your commits. (I'll do this cleanup in a separate PR that doesn't contain any code changes)

I have changed to editor to using tabs. After you have cleaned up the format, should I run clang-format before submitting a PR to avoid polluting the format?

Reply:
Thanks for updating the editor! I'll watch it in future PR's, let's now worry about it in this PR.